### DISCUSSION TITLE
Recovery documentation page broken?

### DISCUSSION THREAD

**Discussion by jonnyijapan** [ory/kratos#2566] [Ideas]
If I open this link: https://www.ory.sh/docs/kratos/self-service/flows/account-recovery#recovery-for-server-side-browser-clients

It shows a page that looks like this. It looks like it might be broken, or some design isn't working. It looks like a dark blue background, with text that is dark brown/grey. The text is almost invisible, it's not easy to even spot the text.
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/278883/177181639-ba7caf33-d645-49e9-b87a-69605e632495.png">

If I do a cmd-a to select all text, the text becomes easier to see...
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/278883/177181892-7270273f-e815-45ce-8f3d-f01ac712ead5.png">

**Comment by aeneasr** [comment]
Thank you for the report! tracked here: https://github.com/ory/docs/issues/891
